### PR TITLE
[Trivial] Grammar and typo correction

### PR DESCRIPTION
**Pull request by ghost**
Minor error correction in src\test\* .

### CODE REVIEW

**Comment by paveljanik** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/9543/commits/e25916482dac8b94e3a4c2eb5560241bfb96e378

**Comment by instagibbs** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/9543/commits/e25916482dac8b94e3a4c2eb5560241bfb96e378

File: src/test/limitedmap_tests.cpp [hunk 1]
```diff
@@ -27,7 +27,7 @@ BOOST_AUTO_TEST_CASE(limitedmap_test)
     // make sure that the size is updated
     BOOST_CHECK(map.size() == 1);
 
-    // make sure that the new items is in the map
+    // make sure that the new items are in the map
```
Comment (MarcoFalke) [inline src/test/limitedmap_tests.cpp]: It is only one item, so singular was correct before?

Comment (laudaa) [inline src/test/limitedmap_tests.cpp]: It does indeed look like only a single item was added above. That would make this another correction to '..new item is in the map', correct?

Comment (paveljanik) [inline src/test/limitedmap_tests.cpp]: Yes please.

**Comment by fanquake** [comment]
ACK post nit-fix

**Comment by MarcoFalke** [comment]
Merged via #9610, sorry for the mess.

**Pull request closed without merging**